inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sort/util/ExtractNodeUtils.java

```diff
@@ -246,7 +246,9 @@ public static PulsarExtractNode createExtractNode(PulsarSource pulsarSource) {
                 serviceUrl,
                 format,
                 startupMode.getValue(),
-                primaryKey);
+                primaryKey,
+                "subName",
+                "earliest");
```

Comment:
hard code?

Reply:
wait for https://github.com/apache/inlong/pull/7937 to be merged first